bench_scripts/combo_dec.py

```
@@ -0,0 +1,19 @@
+import os, time, sys
+os.environ.setdefault("VLLM_USE_V1","1")
+from vllm import LLM, SamplingParams
+def main():
+    preset = sys.argv[1]
+    kw = dict(model="QuantTrio/Qwen3.5-9B-AWQ", tensor_parallel_size=2, dtype="float16",
+              enforce_eager=True, gpu_memory_utilization=0.85, max_model_len=4096,
+              **{"language_model_only": True})
+    if preset!="auto": kw["kv_cache_dtype"]=preset
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)